app/assets/javascripts/discourse/components/topic-timer-info.js.es6

```diff
@@ -18,6 +19,9 @@ export default Ember.Component.extend(
     buildBuffer(buffer) {
       if (!this.get("executeAt")) return;
 
+      const topicClosed = this.get("topicClosed");
+      if (topicClosed != null && (topicClosed ? "close" : "open") === this.get("statusType")) return;
```

Comment:
`!=` is not valid should be `!=='

But in this case:

```es6
if (topicClosed && (topicClosed ? "close" : "open") === this.get("statusType")) {
  return;
}
```

should be enough

Reply:
Thanks, I have to Google about `!=` vs `!==` almost every single time lol.

Reply:
👀 looking into the failed test.